scripts/get-hypper

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env bash
+
+# Copyright The Helm Authors, SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The install script is based off of the MIT-licensed script from glide,
```

Comment:
We need to maintain the copyrighters of that file then, per MIT license.

Reply:
I wonder why the Helm installer doesnt have it...Im no expert on licenses so you migth be rigth here, let me have a look. 

To be fair, this one is based of the helm installer which is licensed under apache....so Im not sure what to do.

Reply:
done!